### PR TITLE
added [News] The Most Innovative Cloud Management Companies

### LINKED ISSUE #3608: [News] The Most Innovative Cloud Management Companies
We're in the news! Using the contributing instructions for [adding news](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-news), please add the following article to the [Layer5 Newsroom](https://layer5.io/news).

**Article:**
- Title: The Most Innovative Cloud Management Companies
- Link: https://beststartuptexas.com/101-texas-united-states-based-cloud-management-companies-the-most-innovative-cloud-management-companies/

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by ShadyResurrected**
**Description**

This PR fixes #3608
Added [News] The Most Innovative Cloud Management Companies

**Notes for Reviewers**

- [ShadyResurrected] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 600235901f9e908041ca39f48e17355d25b6200b at: https://63beb23e9e60f00255bfc77b--layer5.netlify.app

File: src/collections/news/2023/2023-01-11-the-most-innovative-cloud-management-companies/index.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: "The Most Innovative Cloud Management Companies"
+subtitle: ""
```
Comment (leecalcote) [inline src/collections/news/2023/2023-01-11-the-most-innovative-cloud-management-companies/index.mdx:3]:
```suggestion
subtitle: "Layer5 recognized as one of the most innovative cloud management companies"
```

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks, @ShadyResurrected. We'll need a different image for this news article.

**Comment by l5io** [comment]
🚀 Preview for commit e0b0a390eaa89fe39215afd9d86f93332434f76c at: https://63befaa9ec760c09b811133b--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b39f535d64323321345cd9e4831319a479daa54d at: https://63bf052d423d1e1ab19f2dcd--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b39f535d64323321345cd9e4831319a479daa54d at: https://63c0c41da3f71f31e7965e2e--layer5.netlify.app

File: src/collections/news/2023/2023-01-11-the-most-innovative-cloud-management-companies/index.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+date: 2023-01-11 15:58:36 +0000
+author: Best Startup Texas
+thumbnail: ./Best_startup_texas.jpeg
+darkthumbnail: ./Best_startup_texas.jpeg
+eurl: https://beststartuptexas.com/101-texas-united-states-based-cloud-management-companies-the-most-innovative-cloud-management-companies/
+published: false
```
Comment (leecalcote) [inline src/collections/news/2023/2023-01-11-the-most-innovative-cloud-management-companies/index.mdx:9]:
```suggestion
published: true
```

**Comment by leecalcote** [comment]
Deploy preview building...

**Review by leecalcote — APPROVED** [review]
Thank you, @ShadyResurrected. This looks great. 👍

**Pull request merged**